src/cascade/core/db.py

```diff
@@ -93,6 +93,7 @@ def connection(execution_context=None, database=None):
     connection = ezfuncs.get_connection(database)
     yield connection
     connection.commit()
+    connection.close()
```

Comment:
OMG, how did we miss this? This line was half the point of this context manager.